internal/config/server.go

```diff
@@ -119,7 +119,7 @@ func (c *CloudAuthenticationConfig) validate() error {
 func (c *CloudConfig) setDefaults(v *viper.Viper) error {
 	v.SetDefault("server.cloud", map[string]any{
 		"enabled": false,
-		"address": "flipt.cloud",
+		"address": "https://flipt.cloud",
```

Comment:
The tunnel will actualy need this address without the https scheme, as it will bind to address on QUIC over UDP, as opposed to HTTP. HTTP is used afterwards one the tunnel is reversed. We either need to strip the scheme for starting the tunnel or add the scheme wherever this is then being used for other HTTP purposes.